hibernate-jcache/hibernate-jcache.gradle

```diff
@@ -4,12 +4,16 @@ apply from: rootProject.file( 'gradle/published-java-module.gradle' )
 
 
 dependencies {
-    implementation project( ':hibernate-core' )
-    implementation libraries.jcache
+    api project( ':hibernate-core' )
+    api libraries.jcache
 
     testImplementation project( ':hibernate-testing' )
     testImplementation libraries.mockito
     testImplementation libraries.mockito_inline
 
-    testRuntimeOnly libraries.ehcache3
+    testRuntimeOnly(libraries.ehcache3) {
+        capabilities {
```

Comment:
This block should really be colocated with the version in the libraries.gradle. Whether that is worth the re-engineering required is not something I can judge though.

Reply:
libraries.gradle simply defines the versions we want to use.  It does not apply any dependencies